airbyte-integrations/connectors/source-shopify/source_shopify/schemas/customer_journey_summary.json

```diff
@@ -17,175 +17,179 @@
-        "properties": {
-          "id": {
-            "description": "A globally-unique ID.",
-            "type": ["null", "integer"]
-          },
-          "landing_page": {
-            "description": "URL of the first page the customer landed on for the session.",
-            "type": ["null", "string"]
-          },
-          "landing_page_html": {
-            "description": "Landing page information with URL linked in HTML. For example, the first page the customer visited was",
-            "type": ["null", "string"]
-          },
-          "occurred_at": {
-            "description": "The date and time when the customer's session occurred.",
-            "type": ["null", "string"],
-            "format": "date-time"
-          },
-          "referral_code": {
-            "description": "Marketing referral code from the link that the customer clicked to visit the store.",
-            "type": ["null", "string"]
-          },
-          "referrer_url": {
-            "description": "Webpage where the customer clicked a link that sent them to the online store.",
-            "type": ["null", "string"]
-          },
-          "source": {
-            "description": "Source from which the customer visited the store, such as a platform (Facebook, Google), email, direct, a website domain, QR code, or unknown.",
-            "type": ["null", "string"]
-          },
-          "source_type": {
-            "description": "Type of marketing tactic.",
-            "type": ["null", "string"]
-          },
-          "source_description": {
-            "description": "Describes the source explicitly for first or last session.",
-            "type": ["null", "string"]
-          },
-          "utm_parameters": {
-            "description": "A set of UTM parameters gathered from the URL parameters of the referrer.",
-            "type": ["null", "object"],
-            "properties": {
-              "campaign": {
-                "description": "The name of a marketing campaign.",
-                "type": ["null", "string"]
-              },
-              "content": {
-                "description": "Identifies specific content in a marketing campaign. Used to differentiate between similar content or links in a marketing campaign to determine which is the most effective.",
-                "type": ["null", "string"]
-              },
-              "medium": {
-                "description": "The medium of a marketing campaign, such as a banner or email newsletter.",
-                "type": ["null", "string"]
-              },
-              "source": {
-                "description": "The source of traffic to the merchant's store, such as Google or an email newsletter.",
-                "type": ["null", "string"]
-              },
-              "term": {
-                "description": "Paid search terms used by a marketing campaign.",
-                "type": ["null", "string"]
+        },
+        "customer_order_index": {
```

Comment:
Does that mean that the `customer_order_index` was not nested enough? Does that also mean that those fields are empty in the destination right now as we populate `customer_journey_summary. customer_order_index` instead of `customer_order_index`?

The same question applies to the other fields like `first_visit`

Reply:
It was not nested under the `properties` - yes, but originally it was nested correctly under the `customer_journey_summary` object, as expected. 

![Screenshot 2024-08-06 at 19 22 45](https://github.com/user-attachments/assets/1c816ee5-3d52-4ae2-b47b-32f4936525e2)



I don't think so; no one has asked questions yet.

Reply:
Oh! I got confused by the indentation. So we assume here the if there are fields under a property that does not have `properties`, than the platform use those fields as properties? Or does the platform ignore nested fields?

Like, on master we have this:
```
    "customer_journey_summary": {
      "ready": {
        "description": "Whether the attributed sessions for the order have been created yet.",
        "type": ["null", "boolean"]
      },
```

Either the platform assume `customer_journey_summary` to be an object and does not clean the nested fields in `customer_journey_summary` or the platform just doesn't even consider the nested fields as `customer_journey_summary` is an object. Does that make sense to you?

Reply:
> So we assume here the if there are fields under a property that does not have properties, than the platform use those fields as properties? Or does the platform ignore nested fields?

This should be done by the `destination v2`, which should treat the non-typed object as an `object` that has something, thus it's pushed to the destination as an `json-string`, I believe, instead of the serialized `object`.